.travis.yml

```diff
@@ -14,9 +14,13 @@ matrix:
     - python: 'pypy'
       env: TOXENV=pypy
     - python: '2.7'
-      env: TOXENV=du13
+      env:
+        - TOXENV=du13
+        - PYTEST_ADDOPTS = --cov sphinx --cov-config setup.cfg
     - python: '3.4'
-      env: TOXENV=py34
+      env:
+        - TOXENV=py34
+        - PYTEST_ADDOPTS = --cov sphinx --cov-config setup.cfg
```

Comment:
Why do you enable coverages on py34? I think py36 is much better because it's latest release.

Reply:
Done